### ISSUE TITLE
[Supporting the platform – Make Code – Home page ] ‘Micro bit & Microsoft’ logo are getting disappear after applying high contrast white theme under ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3551]
**User Experience:**
Visually impaired user will be impacted severely if ‘Micro bit & Microsoft’ logo does not adapt high contrast(White) theme as they won't be able to take action on that controls and he/she would miss the functionality of such control or element.

**Note**: User credentials should NOT be included in the bug.

**Pre-requisites:**
Enable high contrast mode. Settings->Ease of Access->Color & High contrast

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify that ' Micro bit & Microsoft’ logo controls are getting disappear after applying high contrast white theme or not.

**Actual Result:**
‘Micro bit & Microsoft’ logo are getting disappear after applying high contrast white theme  under ‘Home Page’, due to this user is unable to see the control.

**Expected Result:**
‘Micro bit & Microsoft’ logo controls should be visible after applying high contrast white theme under home page, so that user could easily see the controls.

**WCAG Reference:**
[WCAG 4.3.1 - No Disruption of Accessibility Features](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.3.1, #Win10-Edge(Chromium)

![High contrast white theme](https://user-images.githubusercontent.com/72871089/97856934-0206e800-1d23-11eb-8ae1-3bbdda8d8566.jpg)

**Comment by jwunderl** [comment]
Should be able to detect with `media screen and (-ms-high-contrast: black-on-white) { /*  */ }` I believe and swap to dark logos
